debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleOffsetContext.java

```diff
@@ -36,13 +36,13 @@
      */
     private boolean snapshotCompleted;
 
-    public OracleOffsetContext(OracleConnectorConfig connectorConfig, long scn, Long commitScn, LcrPosition lcrPosition,
+    public OracleOffsetContext(OracleConnectorConfig connectorConfig, String scn, String commitScn, LcrPosition lcrPosition,
```

Comment:
Not `Scn` here instead of string?

Reply:
It can be, oversight during my refactor.